pkg/appmanager/as3Manager.go

```diff
@@ -587,15 +587,14 @@ func (appMgr *Manager) getCertFromConfigMap(cfgmap string) {
 
 	certificates = ""
 	namespaceCfgmapSlice := strings.Split(cfgmap, "/")
-	if len(namespaceCfgmapSlice) < 2 {
-		log.Debugf("[AS3] Invalid trusted-certs-cfgmap option provided.")
+	if len(namespaceCfgmapSlice) != 2 {
+		log.Debugf("[as3_log] Invalid trusted-certs-cfgmap option provided.")
 	} else {
 		certs := ""
-		namespace := namespaceCfgmapSlice[0]
-		cfgmapName := namespaceCfgmapSlice[1]
-		cm, err := appMgr.kubeClient.CoreV1().ConfigMaps(namespace).Get(cfgmapName, metaV1.GetOptions{})
+		cm, err := appMgr.getConfigMapUsingNamespaceAndName(namespaceCfgmapSlice[0], namespaceCfgmapSlice[1])
 		if err != nil {
-			log.Debugf("[AS3] Reading certificate from configmap error: %v", err)
+			log.Errorf("[AS3] ConfigMap not found for this name: %v and namespace: %v, error: %v",
```

Comment:
can be, 
ConfigMap '%v' not found in namespace '%v'. Error: %v.

Reply:
Done